### ISSUE TITLE
Frequency reader is not precise

### ISSUE THREAD

**Issue by Essay97** [seven-phases/spectrum-analyzer#3]
When I put the mouse over a certain frequency, the info on top right corner shows another quantity. For example, if you try to hover the "200Hz" line, on top right you will read "230Hz". I am using 1.09 version, x64 on Windows 10 in Ableton Live Lite 10

**Comment by matdombrock** [comment]
I can also confirm this problem is present in reaper.

**Comment by seven-phases-max** [comment]
Yes.  I continuously suffer from this bug myself :( (I guess it is addressed with https://github.com/seven-phases/spectrum-analyzer/commit/b3638de0a194cf43afb5e96c9e6bc7c9ca2687d7 - just a few days *after* 1.09 release - but I did not manage to release any new binaries after that, sorry)
